### PR TITLE
[OSCD] Create powershell_icmp_exfiltration.yml

### PR DESCRIPTION
**Pull request by bczyz1**
OSCD Sprint 2 Atomic Red Team Windows - #1013 
Task 2 - T1048.003: Exfiltration Over Unencrypted/Obfuscated Non-C2 Protocol

Sigmac Test:
```bash
python tools/sigmac -t splunk -c splunk-windows  rules/windows/powershell/powershell_icmp_exfiltration.yml
(source="WinEventLog:Microsoft-Windows-PowerShell/Operational" EventCode="4104" ScriptBlockText="*New-Object*" ScriptBlockText="*System.Net.NetworkInformation.Ping*" ScriptBlockText="*.Send(*")
```

### CODE REVIEW

**Comment by e6e6e** [comment]
Hello, @bczyz1 ! 
Welcome and thanks for your participation!
I think it's ready to be merged. @yugoslavskiy

File: rules/windows/powershell/powershell_icmp_exfiltration.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: PowerShell ICMP Exfiltration
+id: 4c4af3cd-2115-479c-8193-6b8bfce9001c
+status: experimental
+description: Detects Exfiltration Over Alternative Protocol - ICMP. Adversaries may steal data by exfiltrating it over an un-encrypted network protocol other than that of the existing command and control channel.
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1048.003/T1048.003.md#atomic-test-2---exfiltration-over-alternative-protocol---icmp
+    - https://attack.mitre.org/techniques/T1048/003/
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_icmp_exfiltration.yml:7]: Hello @bczyz1! Good to see you here (:
The rule looks perfect, I would only ask you to remove the link to the ATT&CK techniques from the references, as it is already linked in the `tags`.
Thank you for your contribution!

Comment (bczyz1) [reply rules/windows/powershell/powershell_icmp_exfiltration.yml]: Hey @yugoslavskiy, thanks, it's good to be here ;)
I've removed the redundant ref in b6876e51235f913521235ade5e9a036bb98d7074.

**Comment by yugoslavskiy** [comment]
Thank you for your contribution, @bczyz1! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**